NEWS.md

```diff
@@ -6,6 +6,8 @@
 
 * Corrected confidence interval values for precision components in `tidy.betareg()` output (#1169).
 
+* `tidy.betareg()` will now pass its ellipses `...` to `summary()` internally (#1198 by `@DanChaltiel`). Among other things, this allows choosing the type of residuals.
```

Comment:
I'm a bit confused by the reference to residuals here, as that seems more relevant to predictions with `augment()` than terms with `tidy()`. With this PR (as ought to be the case), passing `residuals` doesn't affect output since `tidy()` doesn't report on residuals:

``` r
library(broom)
library(betareg)
  
data("GasolineYield")
  
fit1 <- betareg(yield ~ batch + temp, data = GasolineYield)

all.equal(
  tidy(fit1, residuals = "deviance"),
  tidy(fit1, residuals = "pearson")
)
#> [1] TRUE
```

<sup>Created on 2024-05-06 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Reply:
As you can see in the test file, the type of residuals can be changed using the `type` argument, not `residuals`. I assumed this is the most common argument passed to `summary()`, but I admit it is only based on my own experience.

Reply:
Ah, my mistake! Same goes for the argument you'd referred to, though:

``` r
library(broom)
library(betareg)

data("GasolineYield")

fit1 <- betareg(yield ~ batch + temp, data = GasolineYield)

all.equal(
  tidy(fit1, type = "deviance"),
  tidy(fit1, type = "pearson")
)
#> [1] TRUE
```

<sup>Created on 2024-05-09 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Reply:
Indeed, I must have brainfarted on this whole issue as you are right, the output of tidy is totally independent of the residuals type.
The only interest left is to silence the warning, as in the test below. It makes it clear that you cannot test the output, only the warning.
Not sure if this PR is worth it for this sole use...

Reply:
All is well! I totally did as well when I first read the issue. Thanks for the PR—I'll go ahead and close.